imageai/Detection/Custom/__init__.py

```diff
@@ -423,44 +426,31 @@ def evaluateModel(self, model_path, json_path, batch_size=4, iou_threshold=0.5,
                     average_precisions = evaluate(infer_model, valid_generator, iou_threshold=iou_threshold,
                                                   obj_thresh=object_threshold, nms_thresh=nms_threshold)
 
+                    result_dict = {
+                        'model_file': model_path,
+                        'using_iou': iou_threshold,
+                        'using_object_threshold': object_threshold,
+                        'using_non_maximum_suppression': nms_threshold,
+                        'average_precision': dict()
+                    }
                     # print the score
                     print("Model File: ", model_path, '\n')
```

Comment:
print("Model File: ", os.path.join(model_path, model_file), '\n')

Reply:
my apologies for this, I think the error is not only here but also some previous lines, I'm working on that rigth now